L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrelatorLayer1Producer.cc

```diff
@@ -1342,6 +1056,12 @@ std::unique_ptr<l1t::PFCandidateCollection> L1TCorrelatorLayer1Producer::fetchPF
       ret->back().setHwEmID(p.hwEmID);
       ret->back().setCaloEta(reg.floatGlbEtaOf(p));
       ret->back().setCaloPhi(reg.floatGlbPhiOf(p));
+      
+      // encode the PF candidate using the 64b encoding used for PUPPI
+      l1ct::PuppiObj encodedPF;
+      encodedPF.fill(reg, p, p.intPt(), 1);
```

Comment:
```suggestion
      encodedPF.fill(reg, p, p.hwPt, 1);
```
after some debugging totally unrelated to this I noticed that the implementation with `p.intPt()` wasn't correct, as the fill method expects a `ap_fixed`  (the version with intPt() compiles, but the packed pT ends up incorrectly multiplied by 4)